src/components/map/Map.tsx

```diff
@@ -34,7 +34,8 @@ type Props = {
 };
 
 const MARKER_ICON_SIZE = normalize(40);
-const CIRCLE_SIZES = [60, 50, 40, 30];
+const CIRCLE_SIZES = [normalize(60), normalize(50), normalize(40), normalize(30)];
+const HITBOX_SIZE = normalize(80); // Größe der unsichtbaren Hitbox
```

Comment:
no German please. and the comment can be removed, as nobody needs a explanation for hitbox size i guess

Reply:
removed in the upcoming commit.